config-example.yaml

```diff
@@ -14,3 +14,6 @@ notifier:
     enable: false
     api_token: 'dummy_token'
     user_key: 'dummy_key'
+  telegram:
+    bot_token: 'dummy_bot_token'
+    chat_id: 'dummy_chat_id'
```

Comment:
Please add `enable: false` config key.

Reply:
dang, I probably removed it when renaming the keys